test/polygons/sponge.cpp

```diff
@@ -836,7 +836,7 @@ TEST(Polygon, Sponge3a) {
   TestPoly(polys, 454);
 }
 
-TEST(Polygon, BigSponge) {
+TEST(Polygon, DISABLED_BigSponge) {
```

Comment:
I just disabled this for now since it's an existing triangulator issue - I also added a more compact repro of it in `Polygon.SquareHoles2`.